src/game_character.cpp

```diff
@@ -415,9 +410,6 @@ void Game_Character::UpdateMoveRoute(int32_t& current_index, const RPG::MoveRout
 				data()->route_through = false;
 				break;
 			case RPG::MoveCommand::Code::stop_animation:
-				if (IsContinuous()) {
-					SetAnimFrame(RPG::EventPage::Frame_middle);
-				}
```

Comment:
> Animtype fixes 

Are you sure with this deletion? It was added because when "Stop Animation" is called the movement frame is  supposed to reset to middle.
I suppose the new fix is the commit after this one? (currently 09eff58 until a rebase)

Reply:
This was moved to `UpdateAnimation()`. Same comments as the jumping case.